velero/backup/common-service/local.properties

```diff
@@ -0,0 +1,25 @@
+# Change the following values to match your environment
+OPERATOR_NS="" # Pass the namespace where the cs operator is installed
+SERVICES_NS="" # Pass the namespace where the cs services are installed
+CONTROL_NS="" # Pass the control namespace if it is needed to be backed up
+
+# Change to the namespace where cert-manager, licensing and LSR are installed
+CERT_MANAGER_NAMESPACE="ibm-cert-manager" 
+LICENSING_NAMESPACE="ibm-licensing"
+LSR_NAMESPACE="ibm-lsr"
+
+ENABLE_PRIVATE_CATALOG=0 # Change to 1 to enable private catalog
+
+# Change the CatalogSource to the namespace where the CatalogSource is installed
+CS_SOURCE_NS="openshift-marketplace" # CatalogSource namespace for common services
+CM_SOURCE_NS="openshift-marketplace" # CatalogSource namespace for cert-manager
+LIS_SOURCE_NS="openshift-marketplace" # CatalogSource namespace for licensing services
+LSR_SOURCE_NS="openshift-marketplace" # CatalogSource namespace for licensing services report
+EDB_SOURCE_NS="openshift-marketplace" # CatalogSource namespace for EDB
+
+# Change the CatalogSource name that installed
+CS_SOURCE="opencloud-operators"
+CM_SOURCE="ibm-cert-manager-catalog"
+LIS_SOURCE="ibm-licensing-catalog"
+LSR_SOURCE="ibm-license-service-reporter-bundle-catalog"
+EDB_SOURCE="cloud-native-postgresql-catalog"
```

Comment:
Maybe we have an optional variable at the bottom that's something like "Additional catalog sources." Right now if the user wants to include a catalogsource that is not one of the default we don't have a way to facilitate that through the script. Could just be one line, comma or space delimited.